### DISCUSSION TITLE
"Date Picker" field on mobile device

### DISCUSSION THREAD

**Discussion by berma83** [Budibase/budibase#2518] [Help] [answered]
Hello guys!

I'm creating some screens for mobile devices, but I'm not able to select a date with the Date Picker field.

https://user-images.githubusercontent.com/53205779/131741089-7b589a8a-f4ea-488f-b0ec-4bca86631d0d.mp4

Current Version: 0.9.117

**Comment by KernOil** [comment]
Hi, @berma83 I don't think they have any mobile compatibility yet. have you tried running the phone browser in desktop mode?

**Comment by mjashanks** [comment, marked as answer]
Hey @berma83 what Mobile OS and Browser are you using? Thanks.

@AndrewJRamay we do have mobile compatibility. You can reformat your interface using conditional UI and `{{ Device.Mobile }}` / `{{ Device.Tablet }}` bindings.

**Reply by KernOil** [reply]
@mjashanks oh, awesome I look forward to using that new functionality!
